.github/workflows/backport.yml

```diff
@@ -76,11 +79,28 @@ jobs:
               break
             fi
           done
-          matrix=$(jq -nc '{include: $ARGS.positional | map_values({milestone: .})}' --args "${target_milestones[@]}")
+
+          pr_head=$(gh pr view ${{ steps.commit.outputs.pr_number }} --json headRepositoryOwner,headRefName --jq '.headRepositoryOwner.login + ":" + .headRefName')
+          response=$(curl -s -o response.json -w "%{http_code}" -X GET "${{ secrets.KVSTORE_URL }}/?key=head_$pr_head" \
+            -H "Authorization: Bearer ${{ secrets.KVSTORE_TOKEN }}")
+          if [ "$(cat response.json)" == "404" ]; then
+            target_branches=($target_milestones)
+          else
+            pr_base=$(jq -r '.value' response.json)
+            target_branches=("backport/${pr_base}-to-${target_milestones[@]}")
+          fi
```

Comment:
Is there a reason why you checked for `404` instead of `200`?

Reply:
If there is no pr with that branch as head (except if base is main), this means that it is either the bottom of the stack or not a stacked pr.
So if it doesn't exist (404), we create a backport based on the existing release branch.